frontend/app/src/components/helper/BinancePairsSelector.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <v-card v-bind="$attrs">
+    <div class="mx-4 pt-2">
+      <v-autocomplete
+        :items="allMakerts"
+        :filter="filter"
+        :search-input.sync="search"
+        multiple
+        :loading="loading"
+        :disabled="loading"
+        hide-details
+        hide-selected
+        hide-no-data
+        return-object
+        chips
+        clearable
+        :outlined="outlined"
+        :open-on-clear="false"
+        :label="label ? label : $t('blockchain_account_selector.default_label')"
+        :class="outlined ? 'blockchain-account-selector--outlined' : null"
+        item-text="address"
+        item-value="address"
+        class="blockchain-account-selector"
+        :value="selection"
+        @input="input($event)"
+      >
+        <template #selection="data">
+          <v-chip
+            :key="data"
+            v-bind="data.attrs"
+            :input-value="data.selected"
+            :click="data.select"
+            filter
+            close
+            @click:close="data.parent.selectItem(data.item)"
+          >
+            {{ data.item }}
+          </v-chip>
+        </template>
+        <template #item="data">
+          <div
+            class="blockchain-account-selector__list__item d-flex justify-space-between flex-grow-1"
```

Comment:
After the classes update the references to these classes would have to be updated too.